### PR TITLE
Stats - File downloads stats card

### PR DESCRIPTION
**Pull request by planarvoid**
Related FluxC changes - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1352

This PR adds a new card - File downloads - to the granular stats screens. The card shows a list of files and the number of downloads in a DESC order by downloads. We only started recording the data after July 29, 2019 so we show a different empty message when the user selects a date before this date. Everything else should work like the other cards.

The file downloads is not supported on Jetpack sites so we hide it there.

I've removed the `RemoteTests` file because it was completely broken.

Screenshot 2019-08-19 at 10 59 22
Screenshot 2019-08-19 at 10 58 46
Screenshot 2019-08-19 at 10 58 25

To test:
* Go to WP.com site
* Go to DWMY Stats
* Scroll down
* You see the file downloads card
* Select a date before July 29, 2019
* You see the "unsupported" error message
* Go to a Jetpack site
* Go to DWMY stats
* The File downloads card is not visible

Update release notes:

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.

### CODE REVIEW

File: WordPress/src/androidTest/java/org/wordpress/android/ui/stats/RemoteTests.java [hunk 1]
```diff
@@ -1,673 +0,0 @@
-package org.wordpress.android.ui.stats;
```
Comment (0nko) [inline WordPress/src/androidTest/java/org/wordpress/android/ui/stats/RemoteTests.java]: Why was this file deleted?

**Review by 0nko — COMMENTED** [review]
I've tested the WP.com and Jetpack sites with different dates and everything works normally. Nice work, Vojta! 💯 

There's just a minor issue with margin when the error message is displayed (see the screenshot):

image

**Comment by 0nko** [comment]
Also, I just noticed that when I tap on the View more button, I can see the loading animation and when it's done there's just blank screen. When I go back and open the View all stats the data's there:

image

File: WordPress/src/androidTest/java/org/wordpress/android/ui/stats/RemoteTests.java (hunk 1 above)
Comment (planarvoid) [reply WordPress/src/androidTest/java/org/wordpress/android/ui/stats/RemoteTests.java]: I've moved it into a separate PR. I doesn't really belong here but the code was broken because the old stats code was already removed. It's just that we never run/build the connected tests

**Comment by planarvoid** [comment]
@0nko I'll look into the "View all" issue separately. Everything else should be fixed and this should be ready for another check. thanks!

**Review by 0nko — APPROVED** [review]
Looking good! 👍
